README.md

```diff
@@ -36,12 +36,17 @@ build one with its internal builder:
 ```java
 Request request = Request.builder()
         .method("POST")
-        .uriWithQuery(URI.create("https://localhost/service/v2/users"))
-        .body("{ field: \"foo\" }")
-        .header("Content-Type", "application/json")
+        .uriWithQuery(URI.create("https://akaa-baseurl-xxxxxxxxxxx-xxxxxxxxxxxxx.luna.akamaiapis.net/billing-usage/v1/reportSources"))
```

Comment:
I think the builder should permissively accept either an absolute or relative path and deal with either one gracefully.